mst/about-mst.md

```diff
@@ -136,6 +136,33 @@ If you require a higher maximum number of connections, but cannot use connection
 pooling, contact the
 [Timescale support team][timescale-support].
 
+## PostgreSQL transaction ID wraparound
```

Comment:
I feel uncertain about this block's placement on this page. In general, we have a strange mix of highly technical vs. more surface-level, very introductory vs. very detailed information on this page, and I think the page's placement is more suited to introductory, surface-level things. Do we want this block to go in troubleshooting instead?

Reply:
Yes!!! you are right!!! May be troubleshooting is the right place.

Reply:
done!!